### PR TITLE
Add actions rust workflow

### PR DESCRIPTION
**Pull request by alex-eg**
Run linter, then run build and tests

### CODE REVIEW

**Comment by Hairash** [comment]
#11

**Comment by alex-eg** [comment]
> Let write in wiki how to run linter on project

https://github.com/Miol-Mor/battle-game/wiki/Tools

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -8,19 +8,21 @@ on:
...
     steps:
+    # Checkout our repo
     - uses: actions/checkout@v2
+    # Install Rust toolchain with required components
     - uses: actions-rs/toolchain@v1
       with:
         toolchain: nightly
         components: clippy, rustfmt
         override: true
+    # Runt rustfmt check against our code
```
Comment (Hairash) [inline .github/workflows/rust.yml:21]: I suppose ```run```

Comment (alex-eg) [reply .github/workflows/rust.yml]: Yep, indeed.

**Pull request merged**